src/python/turicreate/test/test_object_detector.py

```diff
@@ -241,6 +241,12 @@ def test_predict(self):
         pred0 = self.model.predict(sf[:0])
         self.assertEqual(len(pred0), 0)
 
+    def test_predict_with_invalid_annotation(self):
```

Comment:
This seems like a cherry picked commit from another branch. Bad merge?

Reply:
hmmm it seems like this is picked up from my commit.

Reply:
Quite possibly.  I squashed a bunch of trials down for the PR.  I'm happy to let you merge yours first.

Reply:
I'm still running the gitlab for this PR.
I'll let you know if it is merged.